mistralrs-core/src/utils/mod.rs

```diff
@@ -36,14 +36,79 @@ macro_rules! handle_seq_error_stateaware {
                 use $crate::response::Response;
                 use $crate::sequence::SequenceState;
                 // NOTE(EricLBuehler): Unwrap reasoning: The receiver should really be there, otherwise it is their fault.
-                $seq.responder().send(Response::Error(e.into())).unwrap();
+                $seq.responder().send(Response::InternalError(e.into())).unwrap();
                 $seq.set_state(SequenceState::Error);
                 return;
             }
         }
     };
 }
 
+#[macro_export]
+macro_rules! handle_pipeline_forward_error {
+    ($stage: tt, $fallible:expr, $seq_slice:expr, $pipeline:expr, $label:tt) => {
+        match $fallible {
+            Ok(v) => v,
+            Err(e) => {
+                use $crate::response::Response;
+                use $crate::sequence::SequenceState;
+                use $crate::Engine;
+                use $crate::response::SYSTEM_FINGERPRINT;
+                println!("{} - Model failed with error: {:?}", $stage, &e);
```

Comment:
Could you change this to be a `error!` invocation? I think it is better to use `tracing`.